.github/workflows/remove.yml

```diff
@@ -0,0 +1,115 @@
+name: Template Removal
+
+defaults:
+  run:
+    shell: bash
+
+on:
+  issues:
+    types: [labeled]
+  issue_comment:
+    types: [created]
+
+env:
+  REGISTRY_FILE: registry.example.json
+jobs:
+  parse-issue-form-body:
+    name: Parse Issue Form Body
+    runs-on: ubuntu-latest
+    if: (github.event.label.name == 'remove-template') || (github.event_name == 'issue_comment' && github.event.comment.body == '/remove')
+    outputs:
+      npm-link: ${{ steps.issue-input.outputs.npm-link }}
+      npm-package-name: ${{ steps.issue-input.outputs.npm-package }}
+    steps:
+      - name: Parse Issue Form Body
+        uses: peter-murray/issue-forms-body-parser@v2
+        id: issue-payload
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          issue_id: ${{ github.event.issue.number }}
+          separator: "###"
+          label_marker_start: '"'
+          label_marker_end: '"'
+      - name: Parse Issue Payload
+        id: issue-input
+        run: |
+          npm_package=$(echo '${{ steps.issue-payload.outputs.payload }}' | jq '."Name of NPM package"')
+          echo "::set-output name=npm-package::$npm_package"
+          npm_package_path="https://www.npmjs.com/package/"
+          npm_link="$npm_package_path$npm_package"
+          echo "::set-output name=npm-link::$npm_link"
+  validate-author:
+    name: Check that the submitter of the removal request is the same as the submitter of the submission request
```

Comment:
This name will appear in the action UI, so I would prefer to keep it simpler, something like "Validate author" should work